orchagent/pfcwdorch.cpp

```diff
@@ -239,6 +248,24 @@ void PfcWdOrch<DropHandler, ForwardHandler>::createEntry(const string& key,
     }
 
     SWSS_LOG_NOTICE("Started PFC Watchdog on port %s", port.m_alias.c_str());
+
+    // Start PFC storm action on queues of port
+    if (!queues.empty())
+    {
+        for (const auto &q : queues)
+        {
+            size_t qIdx = stoi(q);
```

Comment:
>stoi [](start = 26, length = 4)

stoi may throw.

Reply:
Don't catch all in production code, you can find the specific exceptions types.

---
In reply to: [257485760](https://github.com/Azure/sonic-swss/pull/794#discussion_r257485760) [](ancestors = 257485760)